### DISCUSSION TITLE
[Q&A] 我是一个初级前端，应该如何才能写出如此结构化的项目？

### DISCUSSION THREAD

**Discussion by CallBoson** [lobehub/lobehub#712] [Development Question] [answered]
### 🧐 问题描述 | Proposed Solution

我尝试阅读lobe-chat的源码，发现自己写的简直一塌糊涂，想问下各路大神如何才能写出这样结构化的代码？
我在自己写项目的时候，很多东西都会乱放，代码很乱。

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
不妨看看我们写的开发指南 -> [LobeChat 技术开发上手指南](https://github.com/lobehub/lobe-chat/blob/main/docs/Development/index.zh-CN.md)

[a comment by an automated account is not shown]
